CharacterGenerator/CharacterGenerator/Bender.m

```diff
@@ -0,0 +1,89 @@
+//
+//  Bender.m
+//  CharacterGenerator
+//
+//  Created by Christian Maldonado on 8/4/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "Bender.h"
+#import "BenderManager.h"
+#import "CharacterTableViewController.h"
+
+@implementation Bender
+
+-(void)animalForColor:(NSString *)color AndTexture:(NSString *)texture{
+    
+    
+    [self colorsAndTexturesToNumbers];
+    
+    NSString *c = [self.colorsToNumbers objectForKey:color];
+    NSString *t = [self.texturesToNumbers objectForKey:texture];
+    
+    NSInteger colorValue = [c integerValue];
+    NSInteger textureValue = [t integerValue];
+    
+    NSInteger value = colorValue * textureValue;
+    //Scorpion
+    if(value == 22 || value == 85 || value == 91 || value == 133){
+        self.animal =  [UIImage imageNamed:@"scorpion"];
+    }
+    //Eagles
+    if(value == 26 || value == 39 || value == 51 || value == 65){
+         self.animal =  [UIImage imageNamed:@"eagle"];
+    }
+    //Great White
+    if(value == 38||value == 57|| value == 55){
+        self.animal =  [UIImage imageNamed:@"great_white"];
+    }
+    //Praying Mantis
+    if(value ==34 || value ==33 || value == 95 || value==77){
+        self.animal =  [UIImage imageNamed:@"praying_mantis-1"];
+    }
+    //Unicorn
+    if(value == 91 || value == 119){
+        self.animal = [UIImage imageNamed:@"Unicorn"];
+    }
```

Comment:
What happens if none of the `values` match? self.animal will be NULL. Will your code still work? Does it make sense to have a default animal?